content/en/docs/concepts/services-networking/service.md

```diff
@@ -777,6 +774,10 @@ spec:
   type: ExternalName
   externalName: my.database.example.com
 ```
+{{< note >}}
+ExternalName can be an IP address, but IP addresses are not resolved by CoreDNS or ingress-nginx because ExternalName
```

Comment:
```suggestion
ExternalName accepts an IPv4 address string, but as a hostname comprised of digits, not as an IP address. ExternalNames that resemble IPv4 addresses are not resolved by CoreDNS or ingress-nginx because ExternalName
```